### DISCUSSION TITLE
Naive Question: Can we use CPU instead of GPU during inference?

### DISCUSSION THREAD

**Discussion by liuyifan22** [haotian-liu/LLaVA#1531] [General]
Hello! I am a newcomer in this field, and is wondering whether cpu can be used during inference. If so, I'd like to build one on my own computer which has no gpus for daily use. Thanks!

**Comment by odarcan** [comment]
Check out llava llamafile 

-ngl 0 runs on cpu entirely

**Reply by liuyifan22** [reply]
Thanks for your reply!
